Fig4_PyGMT_pandas.py

```diff
@@ -0,0 +1,62 @@
+import io
+
+import pygmt
+import pandas as pd
+import requests
+from pygmt.params import Box
+
+url = "https://earthquake.usgs.gov/fdsnws/event/1/query"
+params = {
+    "format": "csv",
+    "starttime": "2000-01-01",
+    "endtime": "2025-10-30",
+    "mindepth": 70,
+    "minlongitude": 91,
+    "maxlongitude": 134,
+    "minlatitude": -22,
+    "maxlatitude": 19,
+    "minmagnitude": 5,
+    "orderby": "magnitude",
+}
+r = requests.get(url, params=params)
+df_eqs = pd.read_csv(io.StringIO(r.text))
+df_eqs = df_eqs[df_eqs["magType"] != "mb"]  # Focus on moment magnitudes

```

Comment:
See the comment above:
```suggestion
df_eqs = df_eqs[
    (df_eqs["longitude"] >= 91) & (df_eqs["longitude"] <= 134)
    & (df_eqs["latitude"] >= -22) & (df_eqs["latitude"] <= 19)
    & (df_eqs["magType"] != "mb")
]
```